implementations/elixir/ockam/ockam_hub/test/hub/service/alias_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Test.Hub.Service.AliasTest do
+  use ExUnit.Case
+
+  alias Ockam.Hub.Service.Alias, as: AliasService
+  alias Ockam.Hub.Service.Echo, as: EchoService
+
+  alias Ockam.Router
+
+  test "alias test" do
+    {:ok, _worker, worker_address} =
+      Test.Hub.Service.AliasTestWorker.start_link(address: {192, 168, 2, 1})
+
+    {:ok, _alias, _alias_address} = AliasService.start_link(address: {192, 168, 2, 2})
+    {:ok, _echo, _echo_address} = EchoService.start_link(address: {192, 168, 2, 3})
+
+    msg = %{onward_route: [worker_address], return_route: [], payload: self()}
```

Comment:
Payload is expected to be binary, although it will not break the local routing if you try to use this code with transports, it will break.